src/ECER.Managers.Registry.Contract/PortalInvitations/Contract.cs

```diff
@@ -21,13 +22,25 @@ public record PortalInvitation(string? Id, string Name, string ReferenceFirstNam
   public string? ApplicantFirstName { get; set; }
   public string? ApplicantLastName { get; set; }
   public string? ApplicationId { get; set; }
+  public IEnumerable<CertificationType>? CertificationTypes { get; set; }
```

Comment:
It is safe to set the default value for enumerables to array.empty, it reduces the amount of null checks you need to do later